app/collins/controllers/IpmiApi.scala

```diff
@@ -41,7 +41,7 @@ trait IpmiApi {
   val IPMI_FORM = Form(
     mapping(
       "username" -> optional(text(1)),
-      "password" -> optional(text(8)),
+      "password" -> optional(text(minLength=5, maxLength=16)),
```

Comment:
@defect yes on earlier versions of IPMI, it truncates to 16 chars if you pass it something too long.  I agree, let's increase this to 20 so it can support IPMI 2.0.